src/screens/LoginPage/LoginPage.test.tsx

```diff
@@ -144,7 +145,7 @@ describe('Talawa-API server fetch check', () => {
       );
     });
 
-    expect(fetch).toHaveBeenCalledWith('http://localhost:4000/graphql/');
+    expect(fetch).toHaveBeenCalledWith(BACKEND_URL);
```

Comment:
Why is this necessary for testing when we will always be testing against localhost?